02_combine_counts/replicate_split/replicate_split.cwl

```diff
@@ -0,0 +1,30 @@
+cwlVersion: v1.2
+class: CommandLineTool
+
+doc: |
+  Groups replicate into input and output replicates
+
+hints:
+  DockerRequirement:
+    dockerPull: alpine:3.16.1
+
+requirements:
+  InitialWorkDirRequirement:
+    listing:
+      - $(inputs.replicates)
+
+baseCommand: ls  # Basically, do nothing
+inputs:
+  replicates:
+    type: File[]
+    inputBinding:
+      position: 1
+outputs:
+  input_replicates:
+    type: File[]
+    outputBinding:
+      glob: "Input*.txt"
+  output_replicates:
+    type: File[]
+    outputBinding:
+      glob: "Output*.txt"
```

Comment:
I don't know if I follow the logic of glob-ing files when I believe you should have files already grouped from the previous steps in the CWL pipeline, don't you? Perhaps I'm missing something but is this necessary? If so, I think the are [ExpressionTool](https://www.commonwl.org/v1.0/Workflow.html#ExpressionTool) which could come handy if here

Reply:
The files from the previous step are in one array of files, and here I'm separating them into two arrays, one with Input replicates and one with output replicates.

Reply:
I'm not sure why I'd use something like ExpressionTool when this works well and is super simple.

Reply:
You have to stage files into the command line tool, glob the files and output them, when in reality all you want is to group them. I bet this can get slow with big files